cardigan/stories/components/Contact/Contact.stories.tsx

```diff
@@ -1,15 +1,25 @@
 import Contact from '@weco/common/views/components/Contact/Contact';
+import ContactV2 from '@weco/common/views/components/Contact/Contact.V2';
 import Readme from '@weco/common/views/components/Contact/README.md';
 import { ReadmeDecorator } from '@weco/cardigan/config/decorators';
 
-const Template = args => (
-  <ReadmeDecorator WrappedComponent={Contact} args={args} Readme={Readme} />
-);
-export const basic = Template.bind({});
-basic.args = {
+const args = {
   title: 'Joe Bloggs',
   subtitle: 'Head of Examples',
```

Comment:
Love that Joe is Head of Examples

Reply:
I THOUGHT THE SAME, it made me laugh. You did that btw 😄